docs/reference/client.md

```diff
@@ -34,9 +34,28 @@ object Client {
 The `Client` and `Scope` environments are required to perform the request and handle the response. For the `Client` environment, we can use the default client provided by ZIO HTTP (i.e., `Client.default`). The `Scope` environment is used to manage the lifecycle of resources such as connections, sockets, and other I/O-related resources that are acquired and released during the request-response operation.
 
 :::note
+The `Scope` returned by this method is responsible for running finalizers associated with an HTTP request. It must be closed (using `ZIO.scoped`) after the body of a request has been collected.
+
 To learn more about resource management and `Scope` in ZIO, refer to the [dedicated guide on this topic](https://zio.dev/reference/resource/scope) in the ZIO Core documentation.
 :::
 
+### "Batched" API
+
+The handling of `Scope` can quickly become cumbersome in cases where we simply want to execute an HTTP request and not handle the lifetime of the HTTP request.
+For this reason, the Client can be converted to a batching client where the HTTP request methods don't have Scope in the environment via the `batched` method.
+It is also possible to execute batched requests
+
+```scala
+object Client {
+  def batched(request: Request): ZIO[Client, Throwable, Response] = ???
```

Comment:
Thinking more about this (good) naming convention: could it make sense to rename `Client.request` to `Client.streaming`?

Reply:
Since this is a widely used method, I ended up keeping `Client.request` and annotated it deprecated so that we can point users to the new methods without breaking their code.

Reply:
I am not sure about the deprecated. Then we have to keep this method in until 4.x
I think we should remove it and write migration docs